google-daemon/usr/share/google/google_daemon/accounts_manager.py

```diff
@@ -80,9 +79,7 @@ def Main(self):
           time.sleep(5)
 
         # Write the etag to pass to parent
-        json_tags = json.dumps(
-            [self.desired_accounts.attributes_etag,
-             self.desired_accounts.instance_sshkeys_etag])
+        json_tags = json.dumps([self.desired_accounts.etag])
```

Comment:
Possibly - if a VM upgrades to this change and that file already exists on the machine, I don't want the script to break.